charter/charter-detecting-unwanted-location-trackers.md

```diff
@@ -48,7 +48,7 @@ The WG is expected to:
 5. Design mechanisms to ensure that devices that do not correctly implement or adhere to the DULT protocol can be detected and excluded from being trackable via crowdsourced location networks.
  * This includes considerations for addressing legacy trackers that cannot update to the DULT protocol.
 
-The WG will not define requirements for interactions between accessory manufacturers and law enforcement. Phones, tablets, watches, and laptops are not considered accessories and are out of scope for this working group.
+The WG will not define requirements for interactions between accessory manufacturers and law enforcement. The focus of the WG will be on solving the use case of detecting small and not easily-discoverable accessories, supporting any functionality that is necessary for identifying and recognizing such accessories.
```

Comment:
@ekr does this help explain why the focus would be on the use cases for small devices? Effectively, this lets the protocol work include things like ways to trigger playing a sound or getting a picture of what the accessory looks like, which isn't necessary for the bike case.

Reply:
I think it depends on whether the tag is removable, right?

Reply:
What depends on being removable? I think the main point is to have the focus of the development be on these kinds of devices, and make sure that if there needs to be a way to fetch a URL to get a picture of the device or a command to trigger a sound, those are scoped in.